versioned/storage/store/src/main/java/org/projectnessie/versioned/storage/versionstore/TypeMapping.java

```diff
@@ -107,7 +107,7 @@ public static ContentKey storeKeyToKey(@Nonnull @jakarta.annotation.Nonnull Stor
       variant = raw.substring(idx1 + 1);
     } else {
 
-      keyElements = new ArrayList<>();
+      keyElements = new ArrayList<>(10);
```

Comment:
Would it be worth counting the number of `1` bytes when we search for `idx2` so as to have an exact size for `keyElements`?

Reply:
I doubt